### ISSUE TITLE
1.1.0 release breaks semver

### ISSUE THREAD

**Issue by smeeckaert** [thephpleague/glide-symfony#15]
Since 1.1.0 require `"league/glide": "^2.0"` it will break any code requiring `"glide-symfony": "^1.0"` that uses `\League\Glide\ServerFactory`.

I think that for an API change this big the version should be at least 2.0.

**Comment by tgalopin** [comment]
Technically speaking, if you use ServerFactory in your code, you should have put `"league/glide": "^1.0"` in your composer.json as well (to avoid relying on an indirect dependency), meaning you wouldn't get the new release. It therefore doesn't break SemVer.

I do however understand it may be inconvenient. For the moment I suggest you to force the Glide version (`"league/glide": "^1.0"`) but if too many people have issues with it, I'll consider fixing this.

**Comment by smeeckaert** [comment]
That's what we did to correct the problem but as always with package managers this packages is a dependancy of an other package we have no control over and when I looked at the main packages that depends on glide-symfony and glide-laravel I could only find a few that required `league/glide` directly.

Now I don't know if any of them uses ServerFactory (I encountered this problem on area17/twill so I can say that this update broke this package for sure) but `glide-symfony` has become a de facto shortcut to `league/glide` for those packages.

The glide documentation specifically states that you don't need to install both the adapter and the `league/glide` package which implies that an update on the adapter will not break the compatibility with `league/glide` if only the adapter is required.
